docs/mutation-data-transcript-annotation.md

```diff
@@ -0,0 +1,85 @@
+## Mutation Data Transcript Annotation 
+
+This document describes how each mutation in cBioPortal gets annotated with a specific gene symbol + protein change. 
+
+### Biological Background 
+
+This section explains the concepts of protein isoforms and transcripts. 
+
+#### What is an isoform? 
+
+From a single gene (string of nucleotides) multiple protein sequences can be formed (string of amino acids). For 
+example: parts of the gene that code for proteins (exons) can be included or excluded. Each of these different proteins 
```

Comment:
excluded ... depending upon xyz?

Reply:
each of these different [resulting] proteins is called ...

Reply:
> excluded ... depending upon xyz?

this process is known as alternative splicing. I left it out originally but added a few more lines about it